File: comp2.py
import logging
import sys
import faulthandler

faulthandler.enable(file=sys.stderr, all_threads=True)

logging.basicConfig(filename="/dev/stdout", level=logging.DEBUG, format="%(message)s")

from ebb.comp import Env

def read(name):
    with open(name) as f:
        return f.read()

env = Env()

env.source(read("ebb.ebb"))
env.call("main-lang", (sys.argv[1], read(sys.argv[2])))

File: comp2.mjs
import {argv} from 'process';
import {readFileSync} from 'fs';
import {Env} from './temper.out/ebb/js/comp.js';

const env = new Env();

env.source(String(readFileSync("ebb.ebb")));
env.call("main-lang", [argv[2], String(readFileSync(argv[3]))]);
